Feature - Utopia PHP Storage

lohanidamodar opened the pull request:
## Bringing Appwrite/Storage into an independent storage library under utopia-php. 

This PR creates an independent Storage library, the source code taken from Appwrite/Storage, so that its easier to extend and maintain Storage options.

lohanidamodar commented:
@eldadfux I am having one issue with test, as we are not in docker, the testRoot and testPath are failing, as they return a absolute path. Need help fixing this.
Do we dockerize running tests?

eldadfux commented:
Do we need the demo.php file? can we remove it?

Please add a readme, you can use one of the other utopia libs as template. Please include some basic code examples showing different capabilities of cloud functions. Beside that all looks good.

lohanidamodar commented:
@eldadfux I believe you mean capabilities of `storage`. Will be working on fixing this

eldadfux commented:
hehe yeh sorry, seems like I need a break from cloud functions :)

lohanidamodar commented:
@eldadfux I have updated Readme file and removed demo.php, not sure if it's needed. I am sure, it will have something similar already in Readme example.

eldadfux commented on README.md:
```diff
@@ -1,2 +1,49 @@
+require_once '../vendor/autoload.php';
+
+use Utopia\Storage\Storage;
+
```
Missing init of new Local device. When we have support for S3 please add that as well, and also mention that we support multiple storage adapters in the description

eldadfux commented on README.md:
```diff
@@ -1,2 +1,49 @@
+
+Utopia Framework requires PHP 7.4 or later. We recommend using the latest PHP version whenever possible.
+
+## Authors
```
Can remove this section...
